components/gitpod-protocol/src/experiments/configcat-server.ts

```diff
@@ -35,6 +35,7 @@ export function getExperimentsClientForBackend(): Client {
         requestTimeoutMs: 2000,
         logger: configcat.createConsoleLogger(LogLevel.Error),
         maxInitWaitTimeSeconds: 0,
+        baseUrl: process.env.CONFIGCAT_BASE_URL,
```

Comment:
This should also change to not need the CONFIGCAT_SDK_KEY env variable, always create the client if the BASE_URL is present